script/run-isolated-tests.sh

```diff
@@ -5,10 +5,17 @@
 APP_SUBPATHS="$(find src -name manifest.json | sed -E 's/\/manifest.json//g' | xargs | sed 's/ /|/g')"
 
 # Match changed files to the app paths determined above
-FILES_CHANGED="$(git diff --name-only master)"
-APP_SUBPATHS_CHANGED="$(echo $FILES_CHANGED | grep -oE "($APP_SUBPATHS)")"
+FILES_CHANGED="$(git diff --name-only HEAD $(git merge-base HEAD origin/master))"
+APP_SUBPATHS_CHANGED="$(echo $FILES_CHANGED | grep -oE "($APP_SUBPATHS)" | uniq)"
 NUM_APPS_CHANGED=$(echo "$APP_SUBPATHS_CHANGED" | wc -l)
 
+echo 'Modified files:'
+echo "$FILES_CHANGED"
+echo
+echo 'Modified app dirs:'
+echo "$APP_SUBPATHS_CHANGED"
```

Comment:
For PRs like this one, this variable will be empty.  I'm not a bash expert, but could we do something like this?

```suggestion
[[ ! -z "$APP_SUBPATHS_CHANGED" ]] && echo "$APP_SUBPATHS_CHANGED" || echo "No apps changed"
```

Reply:
There is some more being done in the "no app changes" conditional, so I will move this log below that block, which will exit once its done.